src/components/Actions.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const Actions = () => {
+    return (
+        <div className="actions">
+        {/* Font Awesome icons */}
+        <i class="far fa-comment"></i>
```

Comment:
class en react es una palabra reservada como javascript ya que nos permiten generar clases entonces no podemos hacer uso de class para las clases de css por lo tanto tenemos que cambiar a className 👀 
```suggestion
        <i className="far fa-comment"></i>
        <i className="fas fa-retweet"></i>
        <i className="far fa-heart"></i>
        <i className="fas fa-share"></i>
```